lib/workers/pr/changelog.js

```diff
@@ -27,14 +27,20 @@ async function getChangeLogJSON(depName, fromVersion, newVersion) {
   }
   const semverString = `>${fromVersion} <=${newVersion}`;
   logger.debug(`semverString: ${semverString}`);
+  let token;
   try {
+    // istanbul ignore if
+    if (process.env.GITHUB_ENDPOINT) {
```

Comment:
Should this be `GITHUB_TOKEN`?

Reply:
No.. we only want to unset it for cases where users have a custom github endpoint (i.e. not defaulting to github.com)

Reply:
Except we never set `GITHUB_ENDPOINT`.

We set `RENOVATE_ENDPOINT` to our internal GHE API URL.

Reply:
`RENOVATE_ENDPOINT` gets parsed into `config.endpoint` which then gets pushed back out to `GITHUB_ENDPOINT`, e.g. https://github.com/renovateapp/renovate/blob/b184d3a217c664f215db079e2dba48542bcf6c99/lib/platform/github/index.js#L61

Reply:
Oh, alright. Thank you!